dist/tools/insufficient_memory/create_makefile.ci.sh

```diff
@@ -43,21 +45,30 @@ BOARDS=
 
 rm "${APP_DIR}/Makefile.ci" 2>/dev/null
 touch "${APP_DIR}/Makefile.ci"
+TMPFILE="$(mktemp)"
 
-for BOARD in $(make  --no-print-directory info-boards-supported -C "${APP_DIR}"); do
+for BOARD in $(EXTERNAL_BOARD_DIRS="" make  --no-print-directory info-boards-supported -C "${APP_DIR}"); do
```

Comment:
hm for some reason this does not work, my list of `EXTERNAL_BOARD_DIRS` still gets iterated.

What works is doing 

    EXTERNAL_BOARD_DIRS= make generate-Makefile.ci

but that's easily forgotten and we end up with internal boards in `Makefile.ci`

Reply:
```suggestion
for BOARD in $(make EXTERNAL_BOARD_DIRS="" --no-print-directory info-boards-supported -C "${APP_DIR}"); do
```

Could you give this a try?

Reply:
nope that doesn't make a difference

Reply:
But, it should! 😢